main.js

```diff
@@ -197,7 +209,11 @@ function showItem(passedId) {
 }
 
 function detailMap(coords) {
-    let dMap = L.map('detail-map').setView(L.latLng(coords[0], coords[1]), 12);
+    console.log('coords: ', coords);
+    console.log('coords lat: ', coords.lat)
+    console.log('coords lng: ', coords.lng)
+    // coords[0] and coords[1] should be coords.lat and coords.lng
+    let dMap = L.map('detail-map').setView(L.latLng(coords.lat, coords.lng), 12);
```

Comment:
this is just something else I noticed when reading through your code, though it doesn't have anything to do with the issue that this pull request is for